learning/basics/task1/AP-input_print.py

```diff
@@ -0,0 +1,12 @@
+name = input("hello! what is your name?")
+
+number_1 = int(input("type first number to multiplication"))
+
+number_2 = int(input("type second number to multiplication"))
+
+print(f"{name}, your result of multiplication for {number_1}, and {number_2} is {number_1 * number_2}")
+
+
+
```

Comment:
Too many blank lines on the end of the file, should be exactly 1 blank line (PEP8), try to follow this rule :)